commands/status.go

```diff
@@ -70,6 +73,7 @@ func newStatusResult(status *agentlocal.Status) *statusResult {
 
 	return &statusResult{
 		PMMAgentStatus: status,
+		PMMVersion:     strings.Split(version.PMMVersion, "-")[0],
```

Comment:
please leave as is, no need to split it

Reply:
Sure, just few minutes. I split it, because in description it was without -xxxx.

Reply:
yeah, -xxxx visible only in dev builds

Reply:
Ok done.